vars/archlinux.yml

```diff
@@ -0,0 +1,3 @@
+---
+locale_conf_path: /etc/locale.conf
```

Comment:
Please use the `__` double underscore prefix for internal variables, i.e. `__locale_conf_path`.

Reply:
I guess that should be consistent in the role and variables such as `locale_localectl_cmd` which is registered in `tasks/main.yml` should follow the same pattern?

I would propose to change the internal variable pattern in a commit separate from this pull request. I would rebase this pull request Agreed?